ProjectLighthouse.Servers.Website/Controllers/Moderator/ModerationRemovalController.cs

```diff
@@ -56,52 +56,58 @@ public async Task<IActionResult> DeleteComment(int commentId, [FromQuery] string
         UserEntity? user = this.database.UserFromWebRequest(this.Request);
         if (user == null) return this.Redirect("~/login");
 
-        CommentEntity? comment = await this.database.Comments
-            .Include(c => c.TargetUser)
+        CommentEntity? comment = await this.database.Comments.Include(c => c.TargetUser)
             .Include(c => c.TargetSlot)
+            .Include(c => c.TargetSlot!.Creator)
```

Comment:
```suggestion
            .ThenInclude(s => s!.Creator)
```
This is a slightly cleaner way of including a nested entity